src/libraries/Microsoft.PowerFx.Core/Public/Config/IBasicServiceProvider.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.PowerFx.Core.Public.Config
+{
+    internal interface IBasicServiceProvider : IServiceProvider
+    {
+        void AddService<T>(T service);
```

Comment:
>AddService [](http://example.com/codeflow?start=13&length=10)

interface shouldn't have both methods. 
Make this on an extension method.

Reply:
This looks so illogic to remove it but I can remove it as we don't technically neeed it